packages/core/src/anchor/ethereum/remote-cas.ts

```diff
@@ -44,6 +47,9 @@ function parseResponse(streamId: StreamID, tip: CID, json: unknown): AnchorEvent
         witnessCar: parsed.witnessCar,
       }
     }
+    if (parsed.status === AnchorRequestStatusName.PENDING) {
+      Metrics.count('cas_request_pending', 1)
```

Comment:
I'm not sure how useful this metric will be in practice.  Because we can get this PENDING response many times for the same request as the polling loop loops over the requests that haven't finished processing.  The other two metrics added in this PR don't have that problem because FAILED and COMPLETED are both terminal states that will stop the polling for that request, but PENDING can show up again and again for the same request.

So there won't really be a way to distinguish the case of having one pending request but the loop has processed that one pending request a thousand times, vs there are a thousand pending requests that have each been processed once.

Reply:
BTW this metric we already have I think will be a better proxy for the number of pending requests in flight on the node, as it tracks the size of the AnchorRequestStore: https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/core/src/store/anchor-request-store.ts#L136

Reply:
Running a goose scenario on this and generating high load to observe if pending does us anything useful, I agree that it might give us something us something misleading since its not a terminal state. but it can give us somewhat of a representation of how many entries are in the pending state during a goose attack and track their growth rate. 
Will hold of merging this until I get some numbers from my Goose Scenario

Reply:
The pending count was not telling us much. It was increasing fairly linearly with time and the increase, increase on the amount of requests in the anchor request store did not have any significant impact on the growth of the pending count.